agent/src/balrogagent/client.py

```diff
@@ -57,13 +55,6 @@ async def request(api_root, path, auth0_secrets, method="GET", data={}, headers=
     cookie_jar = aiohttp.CookieJar(unsafe=os.environ.get("ALLOW_COOKIE_FROM_IP_URL", False))
 
     async with aiohttp.ClientSession(loop=loop, cookie_jar=cookie_jar) as client:
-        # CSRF tokens are only required for POST/PUT/DELETE.
-        if method not in ("HEAD", "GET"):
-            logging.debug("Sending %s request to %s", "HEAD", csrf_url)
-            async with client.request("HEAD", csrf_url, headers=headers) as resp:
-                resp.raise_for_status()
-                data["csrf_token"] = resp.headers["X-CSRF-Token"]
-
         # Workaround for https://bugzilla.mozilla.org/show_bug.cgi?id=1471109
```

Comment:
I _think_ it does (the session and the csrf token are separate things, if I recall correctly), but it could be worth trying removing it. It's easy to verify in stage.

Reply:
I think that makes sense, let's do it!